### ISSUE TITLE
Automate deployment to TEST

### ISSUE THREAD

**Issue by freshjwarren** [bcgov/moodle#40]
Acceptance criteria: On the initiation of the Action, new images in TOOLS are picked up and deployed to TEST, where they are confirmed by manually accessing the TEST url.

**Comment by tanvi0ahuja** [comment]
Warren will look into this, Kunal's off sick

**Comment by kkapoor-fresh** [comment]
As per previous discussions with @warrenchristian1telus this is on HOLD at the moment as the TEST env was being used by active users.

**Comment by warrenchristian1telus** [comment]
@kkapoor-fresh all testing should now be in production. @allanhaggett can you please confirm that all users are out of test, and that we're ready to bring it down to implement CI/CD deployments?

**Comment by will-freshworks** [comment]
@kkapoor-fresh blocked on triple checking that they are no longer using the test env "in production"

**Comment by allanhaggett** [comment]
We have checked with the appropriate people and are clear to proceed with this **after 16:30 PST today**

**Comment by kkapoor-fresh** [comment]
@warrenchristian1telus @will-freshworks @freshjwarren which means as per dicussion we can go back to working on CI/CD pipeline -automation to Moodle TEST, right?@allanhaggett
